test/lib/mayaUsd/fileio/testCacheToUsd.py

```diff
@@ -69,17 +108,17 @@ def setUp(self):
         self.proxyShapePathStr = mayaUsd_createStageWithNewLayer.createStageWithNewLayer()
         self.stage = mayaUsd.lib.GetPrim(self.proxyShapePathStr).GetStage()
 
-    def testCacheToUsd(self):
-        '''Cache edits on a pulled Maya reference prim back to USD.'''
+    def runTestCacheToUsd(self, createMayaRefPrimFn, checkCacheParentFn):
```

Comment:
Test engine function that is called to run both the sibling cache and variant set cache tests.